test/evergreen.yml

```diff
@@ -5882,8 +5880,6 @@ buildvariants:
       distros: amazon2-arm64-large
     - name: format-smoke-test
     - name: format-stress-pull-request-test
-    - name: linux-directio
```

Comment:
This keeps slipping back into the changes, I think when merging develop this goes back in without merge conflicts :/